docs/source/channel_update_tutorial.rst

```diff
@@ -647,7 +607,7 @@ for Org3:
 
```

Comment:
You need to change the package label on the command below to match the deployCC script:
```
 peer lifecycle chaincode package basic.tar.gz --path ../asset-transfer-basic/chaincode-go/ --lang golang --label basic_1
```
update to basic_1.0

Reply:
I didn't change anything here, so I assume another PR did and this was missed?

Reply:
Ran through without the change, and everything worked fine. The versions (default from the CLI vs Script) do not need to align.